src/app/shared/filesystem/file-preview/file-preview.component.html

```diff
@@ -1,4 +1,4 @@
 <h3 tabindex="0">
-  <i class="fas fa-exclamation-circle"></i> Preview is not available at this time. Click the "Download Now" button located above to download this object.
+  <i class="fas fa-exclamation-circle"></i> Please log in to preview file. Click the "Download Now" button located above to download this object.
```

Comment:
I think that this will also show if a user is logged in and clicks on something like a `.vmdk` file of which we don't have a preview. We should have this check if the user is logged in or not and display a message accordingly.